### PR TITLE
Use ChoiceType in Log severity column filter

### PR DESCRIPTION
**Pull request by PululuK**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop 
| Description?  | Use `ChoiceType` in Log severity column filter instaed of  `TextType`. 
| Type?         | improvement 
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | N/A.
| How to test?  | Make sure you can filter by severity using the choice input.Capture du 2020-07-16 20-52-27 (1)

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/20200)

### CODE REVIEW

**Review by Progi1984 — DISMISSED** [review]

**Comment by Progi1984** [comment]
@PululuK Travis is :red_circle: 

https://travis-ci.com/github/PrestaShop/PrestaShop/jobs/361860410#L656

File: src/Core/Grid/Definition/Factory/LogGridDefinitionFactory.php [hunk 1]
```diff
@@ -265,4 +268,14 @@ protected function getGridActions()
...
             );
     }
+
+    private function getSeveritysChoices()
+    {
+        return [
+            $this->trans('Informative only', [], 'Admin.Advparameters.Help') => 1,
+            $this->trans('Warning', [], 'Admin.Advparameters.Help') => 2,
+            $this->trans('Error', [], 'Admin.Advparameters.Help') => 3,
+            $this->trans('Major issue (crash)!', [], 'Admin.Advparameters.Help') => 4,
```
Comment (Matt75) [inline src/Core/Grid/Definition/Factory/LogGridDefinitionFactory.php]: Can be useful to add constants for Severity Levels somewhere

Comment (PululuK) [reply src/Core/Grid/Definition/Factory/LogGridDefinitionFactory.php]: Very good ideia ! Thanks @Matt75

**Review by Progi1984 — APPROVED** [review]

**Review by sowbiba — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Comment by SD1982** [comment]
LGTM Thanks @PululuK !!

**Comment by Progi1984** [comment]
Thanks @PululuK & @SD1982

**Pull request merged**